scargo/certs/certGen.sh

```diff
@@ -19,8 +19,10 @@ set -o pipefail # Exit if pipe failed
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 ACTION_COMMAND=""
 ACTION_PARAM=""
-DEVICE_NAME="new-device"
-OUTPUT_DIR=/repo/out/cert
+ACTION_PARAM2=""
+DEVICE_NAME="iot-device"
+OUTPUT_DIR=/workspace/build/cert
```

Comment:
it will be better to have /workspace/ as a  param i.e. - scargo project dir where gen command was trigger